charts/fission-all/values.yaml

```diff
@@ -53,8 +53,6 @@ enableIstio: false
 ## Logger config
 logger:
   influxdbAdmin: "admin"
-  fluentdImage: fission/fluentd
-  fluentdImageTag: 1.0-rc2
```

Comment:
I think it would be good to keep these and give users the flexibility to use their image/tag?